crates/re_viewport/src/space_view_heuristics.rs

```diff
@@ -142,10 +143,12 @@ fn is_interesting_space_view_at_root(
         return false;
     }
 
+    // TODO(andreas): We have to figure out how to do this kind of heuristic in a more generic way without deep knowledge of re_types.
+    //
     // If there are any images directly under the root, don't create root space either.
     // -> For images we want more fine grained control and resort to child-of-root spaces only.
     for entity_path in &candidate.contents.root_group().entities {
-        if contains_any_image(entity_path, data_store) {
+        if contains_tensor_data(entity_path, data_store) {
```

Comment:
Will this mess things up for non-2D tensors?

Reply:
I think you're right. The old code did this:
https://github.com/rerun-io/rerun/blob/release-0.8/crates/re_viewport/src/space_view_heuristics.rs#L76

Reply:
I'm a bit concerned now. I could do the 2dness check on top and be mostly with the old behavior but this has slight regression potential on the fix from https://github.com/rerun-io/rerun/pull/3583
Well, let's try it out!

Reply:
Actually it would be probably best then to check for Image _and_ SegmentationImage indicators.

Reply:
yes. This seems safest, easiest and passes both repro scripts